packages/indy-vdr/README.md

```diff
@@ -0,0 +1,78 @@
+<p align="center">
+  <br />
+  <img
+    alt="Hyperledger Aries logo"
+    src="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript/aa31131825e3331dc93694bc58414d955dcb1129/images/aries-logo.png"
+    height="250px"
+  />
+</p>
+<h1 align="center"><b>Aries Framework JavaScript Action Menu Plugin</b></h1>
```

Comment:
The readme is still for the action menu pacakges. Can you update all references?